.github/dependabot.yml

```diff
@@ -1,31 +1,124 @@
 version: 2
 updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    allow:
+      - dependency-type: all
+    schedule:
+      interval: "daily"
+    labels:
+      - "cargo"
+      - "dependencies"
+    ignore:
+      - dependency-name: "codespan*"
+      - dependency-name: "libfuzzer-sys"
+      - dependency-name: "bindgen"
+    open-pull-requests-limit: 20
```

Comment:
Are the oncalls okay with bumping this up another 5?

Reply:
That's not the interesting part, this is: 

https://github.com/libra/libra/pull/6806/files/3cf3a7187cc6745fc61aedd3d09c34f2cadb0cd9#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28R6

It will result in at least a dozen new prs for our transitive dependencies.    There is no documentation to the effect, but the default behavior of dependabot v2 from github is to only update rust direct dependencies.   Dependabot V1 had different behavior, and security update support for rust.    V2 dependabot does not.

Reply:
got cha. It'd be good to leave this note in the config since there is no documentation.